bblfsh-dashboard/templates/deployment-with-bblfshd-server.yaml

```diff
@@ -16,17 +17,9 @@ spec:
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         args:
-{{- if .Values.settings.serverAddr }}
-        - -bblfsh-addr={{ .Values.serverAddr }}
-{{- else }}
-        - -bblfsh-addr=localhost:9432
-{{- end }}
+        - -bblfsh-addr={{ .Values.settings.serverAddr }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
-{{- if not .Values.settings.serverAddr }}
-      - name: bblfsh-server
-        image: "{{ .Values.server.image.repository }}:{{ .Values.server.image.tag }}"
-        imagePullPolicy: {{ .Values.server.image.pullPolicy }}
```

Comment:
I think next 4 lines should be deleted too.

Reply:
Definitely resources is duplicated. Nice catch. Thx